### ISSUE TITLE
Is there any difference between the open source version of Oncall and the cloud SaaS version?

### ISSUE THREAD

**Issue by shanmu-raoyunfei** [grafana/oncall#4866]
### What would you like to see!

Is there any difference between the open source version of Oncall and the cloud SaaS version?

### Product Area

Other

### Anything else to add?

Is there any difference between the open source version of Oncall and the cloud SaaS version?

**Comment by roock** [comment]
@shanmu-raoyunfei  Yes, there are differences between the cloud version and the open source version.

Some features (e.g. most integrations, MS Teams chatops) are exclusively available on the cloud version and all of them are marked explicitly in the documentation as cloud only.
